### PR TITLE
localization issue

### PR DESCRIPTION
**Pull request by ajewelbd**
Now, by the grace of almighty Allah, user can use localized entity.
Also with expand feature.

### CODE REVIEW

**Comment by gregorwolf** [comment]
Hi @ajewelbd do you run the tests locally before you push? Also it would be great if we could avoid adding a new data model. Here at cds-pg we stick with the beershop instead of bookshop.

**Comment by ajewelbd** [comment]
Hi @gregorwolf,
Sorry for late reply. I create bookshop data model, because I thought if I modify existing schema, may be that can make side effect to other functionalities.

Hope that you will consider the issue.

Thanks.
Md. Ashraful Alam.

**Comment by ajewelbd** [comment]
Hi @vobu,
Sorry for late reply.
By the grace of almighty Allah, it is now tested locally and it works fine.

Thanks
Md. Ashraful Alam.

**Comment by gregorwolf** [comment]
Hi @ajewelbd,

I understand your motivation for the new entities. But I still want to keep the data model more concise. So I would suggest that you add a localized attribute to the TypeChecks entity.

Best Regards
Greogr

**Comment by gregorwolf** [comment]
Please check the error in the test with Node 14. Node 14 does not support replaceAll.

File: __tests__/lib/pg/_runLocal.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ module.exports = async (model, credentials, app, deployDB = false) => {
     kind: 'postgres',
     dialect: 'plain',
     model: model,
-    credentials: credentials,
```
Comment (gregorwolf) [inline __tests__/lib/pg/_runLocal.js:16]: @vobu can you provide the tip how this change can be avoided?

**Comment by ajewelbd** [comment]
Hi @gregorwolf,
Is anything need to modify?

Thanks
Md. Ashraful Alam

**Comment by vobu** [comment]
thanks for being onboard 🙏 - but `cds-pg` sailed has sailed into the official SAP harbor and become a bigger boat ⛵️😄 
https://github.com/cap-js/cds-dbs/tree/main/postgres
Please continue contributing there!

**Pull request closed without merging**